harmonica/forward/prism_layer.py

```diff
@@ -469,3 +487,44 @@ def to_pyvista(self):
                 for data_var in self._obj.data_vars
             }
         return prism_to_pyvista(prisms, properties=properties)
+
+
+def _discard_thin_prisms(
+    prisms,
+    density,
+    thickness_threshold,
+):
+    """
+    Discard prisms with a thickness below a threshold
+
+    Parameters
+    ----------
+    prisms : 2d-array
+        Array containing the boundaries of the prisms in the following order:
+        ``w``, ``e``, ``s``, ``n``, ``bottom``, ``top``.
+        The array must have the following shape: (``n_prisms``, 6), where
+        ``n_prisms`` is the total number of prisms.
+    density : 1d-array
+        Array containing the density of each prism in kg/m^3. Must have the
+        same size as the number of prisms.
+    thickness_threshold : int or float
+        Minimum prism thickness needed to be included in the forward
+        gravity calculation. Prisms thinner that this threshold will be
+        ignored. Default to 0.
+
+    Returns
+    -------
+    prisms : 2d-array
+        A copy of the ``prisms`` array that doesn't include the thin prisms.
+    density : 1d-array
+        A copy of the ``density`` array that doesn't include the density values
+        for thin prisms).
+    """
+    west, east, south, north, bottom, top = tuple(prisms[:, i] for i in range(6))
+    # Mark prisms with thickness <= threshold  as null prisms
+    thickness = top - bottom
+    null_prisms = thickness <= float(thickness_threshold)
```

Comment:
Considering the description of the `thickness_threshold` parameter, wouldn't be better to mark the discarded prisms that have `thickness < thickness_threshold`?
```suggestion
    # Mark prisms with thickness < threshold  as null prisms
    thickness = top - bottom
    null_prisms = thickness < thickness_threshold
```

Also, I suspect that the float casting is not needed.